### PR TITLE
Tidied up

### PR DESCRIPTION
**Pull request by ElleEss**
* "What's new" proofread and add links
* added TOCs
* removed un-necessary "overviews"

### CODE REVIEW

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
 # What's new in alpha?
 
-If you’ve used the pre-alpha GDK you may be interested in these latest changes.
+If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/whatsnew.md]:
```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

File: SpatialGDK/Documentation/content/map-travel.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-Before reading this topic, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).
-
-> Note: This topic is intended for advanced users only.
+This topic is intended for advanced users only. Before reading this page, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).
 
 <%(Callout type="warn" message="ServerTravel support has not yet been merged but will be coming very soon.")%>
 
-## ClientTravel
-`APlayerController::ClientTravel`
+## `APlayerController::ClientTravel`
```
Comment (joshuahuburn) [inline SpatialGDK/Documentation/content/map-travel.md]: `APlayerController::ClientTravel`  Thought we agreed that titles shouldn't be in back-ticks. Looks strange in improbadoc.

File: SpatialGDK/Documentation/content/whatsnew.md (hunk 1 above)
Comment (m-samiec) [inline SpatialGDK/Documentation/content/whatsnew.md]:
```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

Comment (joshuahuburn) [inline SpatialGDK/Documentation/content/whatsnew.md]:
```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
-- Upgraded the engine fork to support Unreal Engine 4.20.3.
-- Upgraded to the new SpatialOS load balancer, providing fast multi-worker iteration.
-- Removed the incremental build time overhead of using the GDK by implementing dynamic type bindings, avoiding the need to generate certain interop code. You can find more info [here]({{urlRoot}}/content/dynamic-typebindings)
-- Schema generation for all classes is now the default.
-- Clients no longer automatically connect to a deployment when not in PIE, unless you pass it a commandline argument. For example: 
-`"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor-Win64-Debug.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1`
-- Startup actors are now saved into the snapshot.
+- Upgraded the SpatialOS Unreal engine fork to support Unreal Engine 4.20.3.
+- Upgraded to the new SpatialOS load balancer, providing fast multi-[worker]({{urlRoot}}/content/glossary#workers) iteration.
+- Removed the incremental build-time overhead to using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This means you don't have to generate certain interop code.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/whatsnew.md]:
```suggestion
- Removed the incremental build-time overhead of using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This removes the additional code-generation and compile step.
```

File: SpatialGDK/Documentation/content/map-travel.md (hunk 2 above)
Comment (ElleEss) [reply SpatialGDK/Documentation/content/map-travel.md]: I wasn't privy to the convo - I was reverting to your original (had to fudge it for launch to make the link work).

Comment (ElleEss) [reply SpatialGDK/Documentation/content/map-travel.md]: Fixed this and the other title which has the same formatting.

File: SpatialGDK/Documentation/content/whatsnew.md (hunk 1 above)
Comment (ElleEss) [reply SpatialGDK/Documentation/content/whatsnew.md]: Done, ta. Added comma, which is better IMHumbleO.

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
-- Singletons can [now be defined]({{urlRoot}}/content/singleton-actors) as “server only”.
-- Initial limited support for Ability System Component on single server only.
-
+- [Authority]({{urlRoot}}/content/authority) is now based on SpatialOS authority.
+- There is easy switching between native-Unreal networking and SpatialOS networking with a [toolbar checkbox]({{urlRoot}}/content/toolbars#switching-between-native-unreal-networking-and-spatialos-networking).
+- The GDK is an Unreal plugin, so you don't need to symlink `UnrealGDK` directories.
+- Improved [component]({{urlRoot}}/content/glossary#spatialos-component) replication support allows you to attach multiple components of the same type to an Actor (or "[entity]({{urlRoot}}/content/glossary#spatialos-entity)" in SpatialOS).
+- [Cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs) facilitate communication with Actors for which another server is [authoritative]({{urlRoot}}/content/authority).
+- `ClientTravel` allows you to move clients between [deployments]({{urlRoot}}/content/glossary#deployment), regardless of whether these deployments are local or in the cloud.
```
Comment (joshuahuburn) [inline SpatialGDK/Documentation/content/whatsnew.md]: For ClientTravel and ServerTravel we should link to the Map Travel doc.

File: SpatialGDK/Documentation/content/map-travel.md (hunk 2 above)
Comment (joshuahuburn) [reply SpatialGDK/Documentation/content/map-travel.md]: Awesome thanks :D

File: SpatialGDK/Documentation/content/whatsnew.md (hunk 3 above)
Comment (ElleEss) [reply SpatialGDK/Documentation/content/whatsnew.md]: "The overhead to using" is better English. There is an overhead to doing something. I can change it if it makes everyone happier, though.

Comment (m-samiec) [reply SpatialGDK/Documentation/content/whatsnew.md]: I took me a few reads to understand what the sentence meant. But I grew up in Australia.

**Pull request merged**